### ISSUE TITLE
Lab 10 Exerc 2

### ISSUE THREAD

**Issue by NFGodoy** [MicrosoftLearning/20532-DevelopingMicrosoftAzureSolutions#214]
# Module: 10
#### Step: 14

Description of issue

Repro steps:

1. Follow the steps as described.
1. The deployment starts and after sometime I get the failure message
1.The 3 urls for the zip files are probably the reason for the failure.
https://github.com/MicrosoftLearning/20532-DevelopingMicrosoftAzureSolutions/raw/master/Allfiles/Mod10/Labfiles/Contoso.Events.Web.zip
generates a 404 if I request it in the browser.
The zip files are in GitHub, but I don't know what would be the url to get them correctly in the template parameters. The one that is in the current template seems to be wrong.

**Comment by seesharprun** [comment]
You are correct. When we published the lab instructions, we didn't have Starter and Solution folders. The link to the WebDeploy packages then didn't work making the ARM template deployment fail.

We updated each link by adding a ``Starter/`` subdirectory and that fixed the issue.

This has been resolved in version 4.1.25:

https://github.com/MicrosoftLearning/20532-DevelopingMicrosoftAzureSolutions/releases/tag/D-v4.1.25
